### DISCUSSION TITLE
test submission

### DISCUSSION THREAD

**Discussion by PierreHao** [snap-stanford/ogb#385] [General]
How do we know that our submission is successfully tested normally？ Is there any response like "Your submission is OK"?

**Comment by weihua916** [comment]
Yes, you will see something like "Success!" on the screen.

**Reply by PierreHao** [reply]
> Yes, you will see something like "Success!" on the screen.

No accurate test results? But we need some details to check for any bugs in our submission. Maybe I think it can return a result like 0.08** of a subset?

**Reply by weihua916** [reply]
Unfortunately, we did not (and are not planning to) implement such functionality. It is your responsibility to ensure the correctness of your submission.

**Reply by PierreHao** [reply]
got it, thank you for your reply
